nim/src/nim.nim

```diff
@@ -0,0 +1,6 @@
+import jester
+import views/general
```

Comment:
```suggestion
import jester, views/general
```

Reply:
I would've done the other way because it separates the library imports versus the local ones